src/components/Accounts/Actions/Actions.js

```diff
@@ -201,15 +201,28 @@ class Actions extends React.Component {
     return this.props.mutator.accounts.PUT(Object.assign(account, newAccount));
   }
 
+  isTagInfo = (values) => {
```

Comment:
Conventionally, a function with the prefix `is` returns a boolean rather than a string, e.g. `isActive(someUser)`, `isError(someResponse)`, `isJohnMalcovich(JohnCusack)`. A good name for this function might be `tagInfo`, `assembleTagInfo`, etc.